src/main/scala/is/hail/expr/FunctionRegistry.scala

```diff
@@ -1164,6 +1164,32 @@ object FunctionRegistry {
     """,
     "p" -> "Probability")
 
+  register("rpois", { (lambda: Double) => rpois(lambda) },
+    """
+    Returns a random draw from a poisson distribution with parameter ``lambda``. This function is non-deterministic.
```

Comment:
I'd say: Returns a random draw from a Poisson distribution with rate (mean) parameter ``lambda``.

because sometimes people use the scale parameter (1 / rate)